packages/grafana-ui/src/components/Menu/SubMenu.tsx

```diff
@@ -99,6 +99,7 @@ const getStyles = (theme: GrafanaTheme2) => {
       top: 0;
       left: 100%;
       z-index: ${theme.zIndex.dropdown};
+      border: 1px solid ${theme.colors.border.weak};
```

Comment:
Why this border? This is not used in DS dropdown

Reply:
unrelated change, will revert